src/main/webapp/app/exercises/shared/rating/rating.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs/Observable';
+import { Rating } from 'app/entities/rating.model';
+import { SERVER_API_URL } from 'app/app.constants';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class RatingService {
+    private ratingResourceUrl = SERVER_API_URL + 'api/results/';
```

Comment:
You're adding the slash twice, resulting in a 500.

![OnPaste 20200526-104359](https://user-images.githubusercontent.com/7109225/82879846-d4c93900-9f3d-11ea-8d1f-ddc831434c8d.png)